src/components/FormModal.tsx

```diff
@@ -149,7 +146,6 @@ export const FormModal = ({
             }}
             disabled={submitDisabled}
           >
-            {icon || (!submitText && <DeprecatedSaveIcon />)}
```

Comment:
This intentional?

Reply:
Yes, we don't need a save icon in the primary submit button that says "save", IMO

Reply:
nvm... I can think of a case for keeping it.  I'll revert.